doc/releases/release-notes-3.1.rst

```diff
@@ -1123,3 +1123,799 @@ Issue Related Items
+* :github:`43448` - Deadlock detection in ``bt_att_req_alloc`` ineffective when ``CONFIG_BT_RECV_IS_RX_THREAD=n``
+* :github:`43440` - Bluetooth: L2CAP send le data lack calling net_buf_unref() function
+* :github:`43430` - Is there any plan to develop zephyr to mircrokenrel architecture?
+* :github:`43425` - zephyr+Linux+hypervisor on Raspberry Pi 4
+* :github:`43419` - Pull request not updated after force push the original branch
+* :github:`43411` - STM32 SPI DMA issue
+* :github:`43409` - frdm_k64f: USB connection gets lost after continuous testing
+* :github:`43400` - nrf board system_off sample application does not work on P1 buttons
+* :github:`43392` - Bluetooth: ISO: unallocated memory written during mem_init
+* :github:`43389` - LoRaWAN on Nordic and SX1276 & SX1262 Shield
+* :github:`43382` - mgmt/mcumgr/lib: Echo OS command echoes back empty string witn no error when string is too long to handle
+* :github:`43378` - TLS availability misdetection when ZEPHYR_TOOLCHAIN_VARIANT is not set
+* :github:`43372` - pm: lptim: stm32h7: pending irq stops STANDBY
+* :github:`43369` - Use Zephyr crc implementation for LittleFS
+* :github:`43359` - Bluetooth: ASCS QoS config should not fail for preferred settings
+* :github:`43348` - twister:skipped case num issue when use --only-failed.
+* :github:`43345` - Bluetooth: Controller: Extended and Periodic Advertising Link Layer Component Test Coverage
+* :github:`43344` - intel_adsp_cavs25: samples/subsys/logging/syst is failing with a timeout when the sample is enabled to run on intel_adsp_cavs25
+* :github:`43333` - RFC: Bring zcbor as CBOR decoder/encoder in replacement for TinyCBOR
+* :github:`43326` - Unstable SD Card performance on Teensy 4.1
+* :github:`43319` - Hardware reset cause api sets reset pin bit everytime the api is called
+* :github:`43316` - stm32wl55 cannot enable PLL source as MSI
+* :github:`43314` - LE Audio: BAP ``sent`` callback missing
+* :github:`43310` - disco_l475_iot1: BLE not working
+* :github:`43306` - sam_e70b_xplained: the platform will be not normal after running test case tests/subsys/usb/desc_sections/
+* :github:`43305` - wifi: esp_at: shell command "wifi scan" not working well
+* :github:`43295` - mimxrt685_evk_cm33: Hard fault with ``CONFIG_FLASH=y``
+* :github:`43292` - NXP RT11xx devicetree missing GPIO7, GPIO8, GPIO12
+* :github:`43285` - nRF5x System Off demo fails to put the nRF52840DK into system off
+* :github:`43284` - samples: drivers: watchdog failed in mec15xxevb_assy6853
+* :github:`43277` - usb/dfu: upgrade request is not called while used from mcuboot, update doesn't happen
+* :github:`43276` - tests: up_squared:  testsuite tests/kernel/sched/deadline/ failed
+* :github:`43271` - tests: acrn_ehl_crb:  tests/arch/x86/info failed
+* :github:`43268` - LE Audio: Add stream ops callbacks for unicast server
+* :github:`43258` - HCI core data buffer overflow with ESP32-C3 in Peripheral HR sample
+* :github:`43257` - can_sam.c
```

Comment:
This issue is ... not clear/useful

Reply:
I'll remove this one since it was converted to a discussion.